### PR TITLE
Update README.md with snapshot info

### PR DESCRIPTION
**Pull request by langsmith**
Initial draft of snapshot info in Android readme. Feel free to edit and/or let me know if other topics need to be covered

### CODE REVIEW

File: platform/android/README.md [hunk 1]
```diff
@@ -14,6 +14,21 @@ Alright. So, actually, you may be in the wrong place. From here on in, this READ
...
+#### Open project in Android Studio
+
+#### Using the SDK snapshot
+
+Instead of using the latest stable release of the Mapbox Android SDK, you can use the nightly build/"snapshot" or the beta version if there is one available. If you'd like to use a snapshot build,  your Android project's gradle file should
+have -SNAPSHOT appended to the SDK version number. For example `5.2.0-SNAPSHOT` or:
+
+```java
+// Mapbox SDK dependency
+    compile('com.mapbox.mapboxsdk:mapbox-android-sdk:5.2.0-SNAPSHOT@aar') {
```
Comment (cammace) [inline platform/android/README.md]: This can be at the same indention level as the comment above it.

File: platform/android/README.md [hunk 2]
```diff
@@ -14,6 +14,21 @@ Alright. So, actually, you may be in the wrong place. From here on in, this READ
...
 [![](https://www.mapbox.com/android-sdk/images/splash.png)](https://www.mapbox.com/android-sdk/)
 
+#### Open project in Android Studio
+
+#### Using the SDK snapshot
+
+Instead of using the latest stable release of the Mapbox Android SDK, you can use the nightly build/"snapshot" or the beta version if there is one available. If you'd like to use a snapshot build,  your Android project's gradle file should
+have -SNAPSHOT appended to the SDK version number. For example `5.2.0-SNAPSHOT` or:
+
+```java
```
Comment (cammace) [inline platform/android/README.md]: You should also include the required repository for snapshots.

Comment (langsmith) [inline platform/android/README.md]: Not sure what you mean...

File: platform/android/README.md (hunk 1 above)
Comment (langsmith) [inline platform/android/README.md]: Fixed

File: platform/android/README.md (hunk 2 above)
Comment (tobrun) [inline platform/android/README.md]:
```
allprojects {
    repositories {
        jcenter()
        maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
    }
}
```

You need to have above section in your root folder build.gradle to be able to resolve these dependencies ( `maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }`)

Comment (langsmith) [inline platform/android/README.md]: 👍

**Comment by langsmith** [comment]
Comments have been addressed @cammace

File: platform/android/README.md [hunk 3]
```diff
@@ -66,6 +90,13 @@ $ sudo apt-get install -y build-essential curl lib32stdc++6 lib32z1 pkg-config p
 
 #### Open project in Android Studio
```
Comment (tobrun) [inline platform/android/README.md]: On macos, you can execute 'make aproj' this will automatically open the project in AS

**Comment by zugaldia** [comment]
> you can use the nightly build/"snapshot"

Looks like since we migrated to CircleCI our snapshots are built every time a PR lands on `master`. We're gonna need to edit the text to reflect that. Otherwise, this is good to merge once we address @cammace + @tobrun reviews.

File: platform/android/README.md (hunk 2 above)
Comment (langsmith) [reply platform/android/README.md]: Fixed

File: platform/android/README.md (hunk 3 above)
Comment (langsmith) [reply platform/android/README.md]: 'make aproj' instructions added

**Comment by langsmith** [comment]
@cammace & @zugaldia , I believe this is ready to go. Updates the Android README. I've made requested changes from above...

**Comment by tobrun** [comment]
@cammace do you want re-review?

**Pull request merged**